symbolic-symcache/src/writer.rs

```diff
@@ -10,6 +10,7 @@ use symbolic_debuginfo::{DebugSession, FileFormat, Function, ObjectLike, Symbol}
 use watto::{Pod, StringTable, Writer};
 
 use super::{raw, transform};
+use crate::raw::NO_SOURCE_LOCATION;
```

Comment:
What will the user see in Sentry with this?

Reply:
They would see an unsymbolicated frame with a "missing_symbol" status, like this (in JSON):
```json
{
  "status": "missing_symbol",
  "original_index": 2,
  "instruction_addr": "0x100821ae0",
  "package": "…",
  "function": "<redacted>",
  "in_app": true
}
```

Reply:
This happens here in Symbolicator: https://github.com/getsentry/symbolicator/blob/859699b085fe5ca668e9811ba55636f07aba71c6/crates/symbolicator-native/src/symbolication/native.rs#L86-L88

Reply:
I think that's sufficient for the use case that lead to this PR in Slack. Thanks again @loewenheim 💯